trino/client.py

```diff
@@ -864,19 +864,22 @@ def _double_map_func(self):
     def _timestamp_map_func(self, column, col_type):
         datetime_default_size = 20  # size of 'YYYY-MM-DD HH:MM:SS.' (the datetime string up to the milliseconds)
```

Comment:
nit: change this to something self-explanatory like `len('YYYY-MM-DD HH:MM:SS.')`.

BTW years are not guaranteed to be 4 digits only.

Reply:
Done

Reply:
I still see the magic value of 20.

Reply:
Ah you meant computing the value dynamically instead of a hard-coded value and an explanation in the comment. Done